src/main/resources/public/ts/controllers/controller.ts

```diff
@@ -911,5 +911,11 @@ export const SupportController: Controller = ng.controller('SupportController',
 			}
 		};
 
+		$scope.disableCreateTicket = () => {
+			if ($scope.ticket.category == null || !$scope.ticket.subject || !$scope.ticket.description){
+				return true
+			}
+		}
+
```

Comment:
Attention à bien penser aux typage de retour, indentation et point-virgule sur ta fonction 👍 
Et tu peux simplifier :
```suggestion
		$scope.disableCreateTicket = () : boolean => {
			return $scope.ticket.category == null || !$scope.ticket.subject || !$scope.ticket.description);
		}

```
Peut être même que le **$scope.ticket.category == null** est simplifiable par un **!$scope.ticket.category**, mais faut vérifier pour le coup. La différence ce fera sur un cas ou ta variable vaut "" par exemple :
- **"" == null** te renvoie **false**
-  **!""** te renvoie **true**